typescript/sdk/src/core/TestCoreDeployer.ts

```diff
@@ -58,6 +72,39 @@ export class TestCoreDeployer extends HyperlaneCoreDeployer {
     return testIsm as unknown as MultisigIsm;
   }
 
+  // deploy a test IGP instead
+  async deployInterchainGasPaymaster(
+    chain: ChainName,
+    proxyAdmin: ProxyAdmin,
+    _gasOracleContracts: GasOracleContracts,
+    deployOpts?: DeployOptions,
+  ): Promise<
```

Comment:
potentially don't need this if you override the `igp` of `coreFactories` in `testCoreFactories`